.nvmrc

```diff
@@ -1 +1 @@
-12.11.1
\ No newline at end of file
+14.19.0
```

Comment:
Could we make this v15.14.0 (the most recent version that is compatible with Vue.js version 2)?

Reply:
NO PLEASE DO NOT! Odd-numbered major releases of Node should *never* be used in production and are ONLY meant for testing upcoming features in Node and ABI changes (for packages that use ABI, and this does not).

Reply:
(If you're curious, this versioning scheme was based on Debian Linux, which does the same thing. In my experience, a LOT of people make this same mistake as a result.)

Reply:
Makes sense, I'll downgrade locally as well!